NuCypher_Beachhead.ipynb

```diff
@@ -0,0 +1,606 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ALICE.network_bootstrap([(\"localhost\", 3601)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### *Bob*: the designated recipient\n",
+    "Bob is the second character in the data sharing narrative. Conceptually, his role is fairly passive – he is simply the chosen recipient. However, as we shall see, there are a number of more complex actions/requirements this character must perform/fulfill in order to securely gain access to the data. Remember, like Alice, Bob represents the device(s) a real-world data recipient – or in our hypothetical application, the doctor – would use to achieve their goals. \n",
+    "\n",
+    "Right now, we'll just instantiate Bob, so we can add him to Alice's sharing policy in the next section.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BOB = Bob()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating a sharing policy with *Alice*\n",
+    "It's time to create our first sharing policy, using the .*grant()* function. Sharing policies have three distinguishing features, which are defined when a policy is granted: \n",
+    "\n",
+    " 1) The data controller, *Alice*.\n",
+    " \n",
+    " 2) The designated recipient, *Bob*.\n",
+    " \n",
+    " 3) The file(s) location, *label*.\n",
+    "\n",
+    "In theory we could leave the *label* argument empty at this stage. However, this would mean ceding important security benefits. For example, in the (highly unlikely) situation where the data recipient successfully colludes with all the Ursulas involved in this sharing narrative, they would only gain access to data kept under that single label in this policy, and nothing else belonging to the data controller, or patient. In this way, labels afford us reliable *forward-secrecy*. In addition, labels are a convenient way to choose specific, per-category sharing parameters for specific data.\n",
+    "\n",
+    "##### Relevance to example real-world application\n",
+    "\n",
+    "In our hypothetical real-world application, the granting of a policy is likely to be the first instance in this narrative where the medical patient proactively does something: namely, deciding to add a specific doctor to an approved list for her health records. Note that Alice does not need to encrypt or send the actual data at this moment. In fact, the data doesn't even need to exist yet – for example, not-yet-available results from an medical exam could be shared automatically under this policy (to all the doctors added to it), once they are ready. More on this scenario below, when we introduce *DataSource*.\n",
+    "\n",
```

Comment:
" In fact, the data doesn't even need to exist yet " --> This line is killer! :sunglasses: